### ISSUE TITLE
Hangs on boot in Portrait (flipped)

### ISSUE THREAD

**Issue by ateich** [Rise-Vision/player-chromeapp#49]
When you set the screen orientation in displays.risevision.com to Portrait (flipped), the RiseVision app hangs on "Initializing application...".

This occurs on an ASUS Chromebox running in Kiosk mode, on reboot.

This issue does not occur when setting the orientation to Portrait or Landscape.

**Comment by mfisher11** [comment]
Ateich,

Can you provide a link to the presentation that you're seeing this issue with?  I've been unable to reproduce it on any of our test devices.

Thanks,

Matthew

**Comment by ateich** [comment]
Hi Matthew,

Here is a link to the presentation preview: http://preview.risevision.com/Viewer.html?type=presentation&id=af4490fd-32be-4d40-a7be-4b7870a54236

Thanks for your help.

Andrew

**Comment by btfrise** [comment]
Hi Andrew,

We were not able to reproduce this issue locally. Can you try recovering your Chromebox to factory settings and let us know if that takes care of it?

**Comment by ateich** [comment]
TLDR; The restores didn’t help, but checking a setting in the chrome extension (“Allow access to file URLs”) did.

I restored the Chromebox to factory settings twice. Once via power washing, the other time using a recovery SD card that had the factory default image from Google on it. Both times the issue occurred.

So I went back into Chrome and was testing different settings for the extension. When I check the box marked “Allow access to file URLs”, which is by default unchecked, the app worked perfectly. Checking this option fixed both of my Chromeboxes that were having the issue. I’m not sure why the other orientations worked with this unchecked, but I'm happy they all do now.

Thanks for all your help.
